pkg/workflows/workload/create.go

```diff
@@ -17,17 +18,17 @@ type Create struct {
 	gitOpsManager    interfaces.GitOpsManager
 	writer           filewriter.FileWriter
 	eksdInstaller    interfaces.EksdInstaller
-	clusterCreator   interfaces.ClusterCreator
+	clusterCreator   clustermanager.ClusterCreator
```

Comment:
why are we changing this?

Reply:
tbh I don't remember why I changed it. I was probably just testing something. changed it back.